share/man/man1/intro.1

```diff
@@ -46,32 +49,57 @@ compilers and compiler tools,
 formatted output tools,
 and line printer commands.
 .Pp
+Tens of thousands of additional commands are available to be installed
+with
+.Xr pkg 8 , or compiled with the
+.Xr ports 7 collection.
+Some of which include web browsers, office suites, calendars,
+conferencing utilities, integrated development environments,
+media players, audio and video processing suites, etc.
+.Pp
 All commands set a status value upon exit which may be tested
 to see if the command completed normally.
 Traditionally, the value 0 signifies successful
 completion of the command, while a value >0 indicates an error.
 Some commands attempt to describe the nature of the failure by using
 exit codes as defined in
 .Xr sysexits 3 ,
-while others simply set the status to an arbitrary value >0 (typically 1).
+while others simply set the status to an arbitrary value >0
+.Pq typically 1 .
+.Sh FILES
+.Bl -tag -width "/usr/local/bin/tab" -compact
```

Comment:
I thought the formatted table looked a lot better in a standard console with 3 extra spaces, which is like a tab.

Reply:
Got it, I don't disagree. There is an official `style.mdoc(5)` rule:
```
   Lists
     - The -width argument to the .Bl macro should match the length of the
       longest item in the list, e.g.:

             .Bl -tag -width "-a address"
             .It Fl a Ar address
             Set the address.
             .It Fl v
             Print the version.
             .El

       In case the longest item is too long and hurts readability, the
       recommendation is to set the -width argument to ‘indent’, e.g.:

             .Bl -tag -width "indent"
             .It Cm build
             Build the port.
             .It Cm install
             Install the port.
             .It Fl install-missing-packages
             Install the missing packages.
             .El


```